src/TMLeague/wwwroot/data/leagues/sl/seasons/s8/divisions/q1.json

```
@@ -19,6 +19,53 @@
     290189,
     290338
   ],
-  "penalties": [],
+  "penalties": [
+      {
+      "player": "PawcioF",
+      "game": 290240,
+      "points": 5,
+      "details": "talking in the open chat"
+    }
```

Comma is needed here!
```suggestion
    },
```